frontend/ui/elements/page_turns.lua

```diff
@@ -26,27 +26,57 @@ end
 genTapZonesMenu("default")
 genTapZonesMenu("left_right")
 genTapZonesMenu("top_bottom")
+
+local default_size_b = math.floor(G_defaults:readSetting("DTAP_ZONE_BACKWARD").w * 100)
+local default_size_f = math.floor(G_defaults:readSetting("DTAP_ZONE_FORWARD").w * 100)
+local function readTapZonesSize()
```

Comment:
(We would usually name it `getTapZonesSize()`, `get` rather than `read`.)

Reply:
Ah yes, agreed.